cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -768,6 +771,244 @@ func TestEvaluateExecute(t *testing.T) {
 		})
 	})
 
+	t.Run("Runtime", func(t *testing.T) {
+		// Skip containerized runtime tests if special cases.
+		if osutil.IsWindows() {
+			t.Skip("Docker runtime not supported on Windows")
+		} else if _, err := exec.LookPath("docker"); err != nil {
+			t.Skip("Docker runtime not found")
+		}
+
+		// Prerequisite
```

Comment:
Extend docstring to that we pull the image of the current branch.

Could we not just build the image here to be 100% sure it is the right one?

Reply:
The CI is built that docker build is a preqreuisite step before the tests run. I am not sure if we should put a docker image build into the testing!?

Reply:
ok... if we 100% have the correct image here than it's fine... just afraid of having some networking hickups and then this image is some previous one